packages/strapi-plugin-i18n/admin/src/components/ModalEdit/BaseForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label } from '@buffetjs/core';
+import { Inputs } from '@buffetjs/custom';
+import Select from 'react-select';
+import { Col, Row } from 'reactstrap';
+import { useIntl } from 'react-intl';
+import { useFormikContext } from 'formik';
+import { getTrad } from '../../utils';
+
+const BaseForm = ({ options, defaultOption }) => {
+  const { formatMessage } = useIntl();
+  const { values, handleChange } = useFormikContext();
+
+  return (
+    <Row>
+      <Col>
+        <span id="locale-code">
+          <Label htmlFor="">
```

Comment:
is this `htmlFor` empty value wanted?

Reply:
The `Label` component requires a `htmlFor` (as a mandatory props). In a normal situation, this htmlFor would point to the id of the select input provided by react-select. 

However, it looks that react-select doesn't allow to pass down an id, but allows to pass an `aria-labelledby` props. This props references an "id" that describes it's content. (the id is the one that you see line 18).

So:

- I had to put the htmlFor because of propTypes
- And I'm referencing the label in the "inverse" way by using aria-labelledby 😊